CODEOWNERS

```diff
@@ -0,0 +1,10 @@
+#############################
+### Provenance Implementation
+# Please refrain from adding more code owners than is
+# explicitly necessary for a project, as this requires 
+# a TAG Security lead to manually add new code owners
+
+provenance-implementation/argo/argo-cd.md @crenshaw-dev
```

Comment:
Does @crenshaw-dev agree to maintain this?

Reply:
Right right— still pending response from him due to the fact that I threw this together after midnight.

Reply:
Yep, I can maintain it!